packages/ragbits-document-search/src/ragbits/document_search/cli.py

```diff
@@ -82,3 +82,19 @@ async def run() -> None:
         print_output(entries, columns=columns)
 
     asyncio.run(run())
+
+
+@ds_app.command()
+def ingest(
+    source: Annotated[str, typer.Argument(help="Source pattern")],
+) -> None:
+    """
+    Ingest the elements from a given source to vector store.
+    """
+
+    async def run() -> None:
+        if state.document_search is None:
+            raise ValueError("Document search not initialized")
+        await state.document_search.ingest(source)
```

Comment:
nit: I don't have strong options on which behavior is better, but the `remove` command from vector-store prints a confirmation message, so now there will be some inconsistency in behavior between those two commands.